docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'OpenCDC record'
+sidebar_position: 2
+---
+
+An OpenCDC record in Conduit aims to standardize the format of data records exchanged between different connectors within a data processing pipeline. The primary objective is to ensure compatibility between various combinations of source and destination connectors.
+
+## Benefits
+
+1. **Support for Operations**: The format should support representing records for create, update, delete, and snapshot operations.
+1. **Standard Metadata Fields**: Definition of standard metadata fields to provide essential information about each record.
+1. **Integration with Data tools**: We believe that by standardizing on this format regardless of the connector type, we'll easily be able to integrate with any other processing data tool that supported this format.
+
+## Fields
+
+* `.Position` uniquely represents the record.
+* `.Operation` defines what triggered the creation of a record. There are four possibilities: `create`, `update`, `delete` or `snapshot`. The first three operations are encountered during normal CDC operation, while `snapshot` is meant to represent records during an initial load. Depending on the operation, the record will contain either the payload before the change, after the change, both or none (see field `Payload`). 
+* `.Key` represents a value that should identify the entity (e.g. database row).
+* `.Metadata` contains additional information regarding the record.
+* `.Payload` holds the payload change (data before and after the operation occurred).
+
+## Representation
+
+Conduit relies on [protobuf](https://protobuf.dev/) when it comes to defining an OpenCDC record to benefit from the several advantages that it provides. Its definition can be found in the [Buf Schema Registry](https://buf.build/conduitio/conduit-commons/docs/main:opencdc.v1).
+
+When processing records in Conduit, you can always expect this format,
+
+```json
+{
+    "position": "c3RhbmRpbmc=",
+    "operation": "update",
+     "metadata": {
+        "file.path": "./example.in",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1"
+    },
+    "key": "cGFkbG9jay1rZXk=",
+    "payload": {
+        "before": "eWVsbG93",
+        "after": {
+            "bool": true,
+            "float32": 1.2,
+            "float64": 1.2,
+            "int": 1,
+            "int32": 1,
+            "int64": 1,
+            "string": "orange"
+        }
+    }
+}
+```
+
+:::note
+
+Depending on the operation performed on the record, and the connector used its payload can vary. For example:
+
+- When operation is `create` or `snapshot`, there won't be a `payload.before`, but a `payload.after` will.
+- When operation is `update`, `payload.after` or `payload.before`, any of these could be present.
+- When operation is `delete`, there won't be a `payload.after`, but there will be a `payload.before`sometimes
```

Comment:
A very small thing: space needed before `sometimes` and also a period is missing at the end. Also, maybe it's good to be consistent when it comes to referencing fields and use `.Payload.After` here.